server/finder/http/server.go

```diff
@@ -47,8 +47,8 @@ func New(listen string, indexer indexer.Interface, registry *providers.Registry,
 	h := newHandler(indexer, registry)
 
 	// Client routes
-	r.HandleFunc("/cid/{cid}", h.GetSingleCid).Methods("GET")
-	r.HandleFunc("/cid", h.GetBatchCid).Methods("POST")
+	r.HandleFunc("/index/{index}", h.find).Methods("GET")
```

Comment:
`/multihash/{multihash}`

We may want other types of queries eventually e.g. selectors, and overloading everything to 'index' doesn't help readability.